From 67464a70d0abb5c4a67c28369e49d112e364862b Mon Sep 17 00:00:00 2001
From: Tom Schraitle <tomschr@users.noreply.github.com>
Date: Mon, 6 Mar 2023 11:42:34 +0100
Subject: [PATCH] Fix #397: Remove asterisk in python_requires

The asterisk in "python_requires = >= 3.7.*" make the
pyproject-build command fail with this exception:

 setuptools.extern.packaging.specifiers.InvalidSpecifier: Invalid specifier: '>=3.7.*'

This fix removes the asterisk which leads to a successful build.
---
 setup.cfg | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.cfg b/setup.cfg
index 2673b0f..0ee8564 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -39,7 +39,7 @@ license = BSD
 package_dir =
     =src
 packages = find:
-python_requires = >=3.7.*
+python_requires = >=3.7
 include_package_data = True
 
 [options.entry_points]
